SonarQube.TeamBuild.PostProcessor/Program.cs

```diff
@@ -29,14 +29,16 @@ private static int Main()
 
             AnalysisConfig config = GetAnalysisConfig(settings, logger);
 
-            logger.Verbosity = VerbosityCalculator.ComputeVerbosity(config.GetAnalysisSettings(true), logger);
-
             if (config == null)
             {
+                LogStartupSettings(config, settings, logger);
                 logger.LogError(Resources.ERROR_MissingSettings);
                 return ErrorCode;
             }
 
+            logger.Verbosity = VerbosityCalculator.ComputeVerbosity(config.GetAnalysisSettings(true), logger);
+            LogStartupSettings(config, settings, logger);
+
```

Comment:
I tweaked the logging of the TeamBuildSettings for the post-processor so verbosity is respected (and the settings won't be dumped in the normal case).